app/views/devise/registrations/new.html.slim

```diff
@@ -1,17 +1,15 @@
-.container
-  .row
-    .col-sm-9.col-md-7.col-lg-7.mx-auto
-      .card
-        .card-block.p-4
-          h2.text-center.p-2
-            = t('.sing_up_header')
-          = simple_form_for resource, as: resource_name, url: registration_path(resource_name), wrapper: :horizontal_form do |f|
-            = f.input :email, placeholder: 'example@example.com', autofocus: true
-            = f.input :password, autofocus: true, hint: 'Minimum 8 characters'
-            = f.input :password_confirmation, autofocus: true
-            = f.input :first_name, autofocus: true
-            = f.input :last_name, autofocus: true
-            = f.input :country, autofocus: true, include_blank: false
-            .d-flex.flex-row-reverse
-              = f.submit 'Sign Up', class: 'btn btn-primary'
-          = render "devise/shared/links"
+.position-block
+  .card
+    .card-block.p-4
+      h2.text-center.p-2
+        = t('.sing_up_header')
+      = simple_form_for resource, as: resource_name, url: registration_path(resource_name), wrapper: :horizontal_form do |f|
+        = f.input :email, placeholder: 'example@example.com', autofocus: true
```

Comment:
Pls use `autofocus: true` only to one input `= f.input :email ...` no need to add to each line.